dashboard/client/src/type/serve.ts

```diff
@@ -0,0 +1,64 @@
+export enum ServeApplicationStatus {
+  // Keep in sync with ApplicationStatus in python/ray/serve/_private/common.py
```

Comment:
Could you add a similar comment linking back to this file in [the `ApplicationStatus` class](https://github.com/ray-project/ray/blob/514bb3abeff8172677f35158746376e8cb4786ca/python/ray/serve/_private/common.py#L31)?